.github/workflows/ubuntu_bionic.yml

```diff
@@ -0,0 +1,41 @@
+name: Ubuntu-Bionic
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  schedule:
+    - cron: '0 5 * * *'
+
+jobs:
+  ci:
+    name: Ubuntu-Bionic
+    runs-on: ubuntu-latest
+    container: ubuntu:bionic
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@master
+        with:
+          path: src/descartes_light
+
+      - name: Install dependencies
+        run: |
+          apt update -q
+          apt install -q -y clang-tidy python3 python3-pip python3-setuptools git
+          pip3 install -q --upgrade pip
+          pip3 install -q colcon-common-extensions rosdep vcstool
+          rosdep init -q
+          rosdep update -q
+          vcs import src < src/descartes_light/dependencies.rosinstall
+          rosdep install --from-paths src --ignore-src -r -y -q
+
+      - name: Build
+        run: |
+          colcon build --event-handlers console_direct+ --cmake-args -DCMAKE_BUILD_TYPE=Debug -DDESCARTES_ENABLE_TESTING=ON -DDESCARTES_ENABLE_CLANG_TIDY=ON
+          if [ $? -ne 0 ]; then exit 1; fi
+
+      - name: Test
+        run: |
+          colcon test --event-handlers console_direct+ --return-code-on-test-failure
```

Comment:
It looks like the option `--return-code-on-test-failure` does not work so it is probably good to add the following after each colcon call.
`if [ $? -ne 0 ]; then exit 1; fi`